public/js/components/ObjectInspector.js

```diff
@@ -131,7 +131,7 @@ const ObjectInspector = React.createClass({
     let objectValue;
     if (nodeHasProperties(item) || nodeIsPrimitive(item)) {
       const object = item.contents.value;
-      objectValue = Rep({ object });
+      objectValue = Rep({ object, mode: "tiny" });
```

Comment:
I'll leave this to you @jasonLaster heh, I'm in the same boat as @linclark though it does seem to look like your expect screenshot.